perfkitbenchmarker/publisher.py

```diff
@@ -106,6 +108,14 @@
     'of all samples as metadata. Multiple key-value pairs may be specified '
     'by separating each pair by commas.')
 
+flags.DEFINE_string(
+    'influx_uri', 'localhost:8086',
+    'The Influx DB endpoint for queries address and port')
```

Comment:
nitpick: The help message readability could be improved, I think you can remove 'endpoint for queries'.

Also, I think it needs clarification on what's the format of the string it expects. It's not clear if passing 'http://hostname:port/' is valid, or if only 'hostname:port' is expected.